pkg/network/nmstate.go

```diff
@@ -169,3 +175,15 @@ func getMinNumberOfWebhookReplicas(clusterInfo *ClusterInfo) int32 {
 
 	return highlyAvailableMinWebhookReplicas
 }
+
+func pupulateDefaultConfiguration(conf *cnao.NetworkAddonsConfigSpec) *cnao.NetworkAddonsConfigSpec {
```

Comment:
This is all at the same package let mark it with "Nmstate"
```suggestion
func pupulateNmstateDefaultConfiguration(conf *cnao.NetworkAddonsConfigSpec) *cnao.NetworkAddonsConfigSpec {
```

Reply:
Ah, good point. [Updated](https://github.com/kubevirt/cluster-network-addons-operator/compare/d0cd05972a7bccd93e2cea4fce9739e169ae7ee6..e4d4ab762341112ecfdb5f7fb749bd52fa3bc833)